components/menu/stories/menu.stories.js

```diff
@@ -471,3 +378,266 @@ TraySubmenu.args = {
+		},
+		{
+			sectionTitle: "Multi-selection with switches",
+			sectionMarkup: MultiCheckboxSelectedStates({...args, selectionMode: "multiple", hasActions: true}),
+		},
+		{
+			sectionTitle: "Multi-selection with switches and switch label",
+			sectionMarkup: MultiCheckboxSelectedStates({...args, selectionMode: "multiple", hasActions: true, items: [{ label: "Menu item", value: "switch label"}]}),
+		},
+		{
+			sectionTitle: "Sizes",
+			sectionMarkup: Sizes(args),
+		},
+	];
+	return sectionData.map((sectionItem) => html`
+		<div class="spectrum-Typography">
+		${Typography({
+			semantics: "detail",
+			size: "l",
+			content: [sectionItem.sectionTitle],
+		})}
+		<div
+			style=${styleMap({
+				display: "flex",
+				flexWrap: "wrap",
+				gap: "1.5rem",
+			})}
+		>
+			${sectionItem.sectionMarkup}
+		</div>
+	</div>
+	`);
+};
+
+export const MenuItemOnly = (args) => window.isChromatic() ?
+	ChromaticMenuItem(args) : Template(args);
+MenuItemOnly.args = {
+	items: [
+		{
+			label: "Hello menu item",
+		},
+	],
+	hasActions: false,
+	hasItemDescription: false,
+	isDisabled: false,
+	isItemActive: false,
+	isItemFocused: false,
+	isItemHovered: false,
+	isItemSelected: false,
+	singleItemDescription: "Menu item description",
+};
+MenuItemOnly.argTypes = {
+
+	isDisabled: {
+		name: "Menu item is disabled",
+		type: { name: "boolean" },
+		table: {
+			type: { summary: "boolean" },
+			category: "State",
+		},
+		control: "boolean",
+	},
+	isItemActive: {
+		name: "Menu item is active",
+		type: { name: "boolean" },
+		table: {
+			type: { summary: "boolean" },
+			category: "State",
+			control: "boolean",
+		},
+	},
+	isItemFocused: {
+		name: "Menu item is focused",
+		type: { name: "boolean" },
+		table: {
+			type: { summary: "boolean" },
+			category: "State",
+			control: "boolean",
+		},
+	},
+	isItemHovered: {
+		name: "Menu item is hovered",
+		type: { name: "boolean" },
+		table: {
+			type: { summary: "boolean" },
+			category: "State",
+			control: "boolean",
+		},
+	},
+	itemIcon: {
+		...(IconStories?.argTypes?.iconName ?? {}),
+		if: false,
+	},
+	isItemSelected: {
+		name: "Menu item is selected",
+		description: "Selected state when single or multi-select mode is turned on",
+		type: { name: "boolean" },
+		table: {
+			type: { summary: "boolean" },
+			category: "Selection",
+			control: "boolean"
+		},
+		if: { arg: "selectionMode", not: { eq: "none" } },
+	},
+	hasItemDescription: {
+		name: "Has menu item description",
+		type: { name: "boolean" },
+		table: {
+			type: { summary: "boolean" },
+			category: "Content",
+			control: "boolean",
+		},
+	},
+	singleItemDescription: {
+		name: "Menu item description for single menu item",
+		type: { name: "text", required: true },
+		table: {
+			type: { summary: "text" },
+			category: "Content",
+		},
+		control: "text",
+		if: { arg: "hasItemDescription", truthy: true },
+	},
+};
+
+export const Default = Template.bind({});
```

Comment:
I added this back in because I did not realize that Picker was using this menu. I'd love to be able to hide this story since it's not needed in the sidebar for Menu but haven't found an example of where we've done that before.